Mathlib/Analysis/SpecialFunctions/Pow/Real.lean

```diff
@@ -420,13 +420,21 @@ theorem rpow_lt_rpow (hx : 0 ≤ x) (hxy : x < y) (hz : 0 < z) : x ^ z < y ^ z :
     exact mul_lt_mul_of_pos_right (log_lt_log hx hxy) hz
 #align real.rpow_lt_rpow Real.rpow_lt_rpow
 
+theorem strictMonoOn_rpow_of_exponent_pos {r : ℝ} (hr : 0 < r) :
```

Comment:
How about:
```suggestion
theorem strictMonoOn_rpow_Ici_of_pos {r : ℝ} (hr : 0 < r) :
```
(and throughout below)

Reply:
I think it would be good to keep "exponent" in the lemma name, to distinguish them from the similar lemmas where the base is fixed and the exponent varies (i.e. [Real.strictMono_rpow_of_base_gt_one](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Analysis/SpecialFunctions/Pow/Real.html#Real.strictMono_rpow_of_base_gt_one) and friends).

Reply:
Fair enough, it's not a big deal.

For theorems we are supposed to name based on the type of the hypotheses / conclusion rather than the semantics but exceptions can be made!